etc/calico/confd/templates/bird.cfg.template

```diff
@@ -1,3 +1,33 @@
+function apply_communities ()
+{
+  {{- $prefix_advertisements_key := ""}}
+  {{- $node_prefix_advertisements_key := printf "/host/%s/prefix_advertisements" (getenv "NODENAME")}}
+  {{- if exists $node_prefix_advertisements_key}}  
+    {{- $prefix_advertisements_key = $node_prefix_advertisements_key}}
+  {{- else if exists "/global/prefix_advertisements"}}
+    {{- $prefix_advertisements_key = "/global/prefix_advertisements"}}
+  {{- end}}
```

Comment:
Right, we haven't discussed that. 
> I am wondering if there could be a use case for someone wanting the global community config PLUS the per-node community config?

Current implementation does node specific override, so if someone wants global plus per-node communities, they should include both in node specific configuration.
But if we make it the other way, where instead of overriding global community config, per-node config gets appended with global community config, then it will be hard to handle the case where someone wants to override global community, we will have to introduce a new field where user allows/denies override to global config.

Reply:
Thanks, it's a strong point that we would have to introduce something extra to allow overriding the global config.  I'm happy that that settles the argument for the current position.